.github/workflows/release.yml

```diff
@@ -56,8 +101,63 @@ jobs:
           git commit -a -m "Update changelog"
           git push "https://${{ env.CI_COMMIT_AUTHOR }}:${{ secrets.TF_DEVEX_COMMIT_GITHUB_TOKEN }}@github.com/${{ github.repository }}.git"
 
+  cdktfDocs:
+    needs:
+      - releasedVersion
+      - buildBinary
+      - changelog # So the commit is pushed for the changelog
+    runs-on: ubuntu-latest
+    container:
+      image: docker.mirror.hashicorp.services/hashicorp/jsii-terraform
+    env:
+      CHECKPOINT_DISABLE: "1"
+    timeout-minutes: 120
+    steps:
+      - uses: actions/checkout@v3.1.0
+      - run: git config --global user.email "bcroft@hashicorp.com"
+      - run: git config --global user.name "Brandon Croft"
```

Comment:
This is handled below (or should definitely be adjusted to use the service account)

```suggestion
```

Reply:
Yeah for sure, In missed that part 🙈